tap-agent/src/agent/sender_account.rs

```diff
@@ -1082,20 +1097,21 @@ pub mod tests {
         next_unaggregated_fees_value: Arc<Mutex<u128>>,
         receipts: Arc<Mutex<Vec<NewReceiptNotification>>>,
     }
-
+    type RavRequestResultTuple = (Arc<AtomicU32>, Arc<Mutex<u128>>);
     impl MockSenderAllocation {
-        pub fn new_with_triggered_rav_request() -> (Self, Arc<AtomicU32>, Arc<Mutex<u128>>) {
+        pub fn new_with_triggered_rav_request() -> (Self, Result<RavRequestResultTuple, ()>) {
             let triggered_rav_request = Arc::new(AtomicU32::new(0));
             let unaggregated_fees = Arc::new(Mutex::new(0));
+            let mock_rav_request_response =
+                (triggered_rav_request.clone(), unaggregated_fees.clone());
             (
                 Self {
-                    triggered_rav_request: triggered_rav_request.clone(),
+                    triggered_rav_request,
                     receipts: Arc::new(Mutex::new(Vec::new())),
                     next_rav_value: Arc::new(Mutex::new(0)),
-                    next_unaggregated_fees_value: unaggregated_fees.clone(),
+                    next_unaggregated_fees_value: unaggregated_fees,
                 },
-                triggered_rav_request,
-                unaggregated_fees,
+                Ok(mock_rav_request_response),
```

Comment:
Why are we returning a `Result` here? I couldn't an error we're trying to handle 🤔

Reply:
Did so to make it more like the [real response](https://github.com/graphprotocol/indexer-rs/blob/f6ebdd8e1b1441e927cc202d9510ff8edba652c7/tap-agent/src/agent/sender_account.rs#L220), where it also responds with a result

Reply:
yeah removed it afterall